source/agora/common/Types.d

```diff
@@ -97,6 +97,19 @@ public struct Height
     {
         return Height(this.value++);
     }
+
+    /// Allow to offset an height by a fixed number
+    public Height opBinary (string op : "+") (ulong offset) const
+    {
+        return Height(this.value + offset);
+    }
+
+    /// Allow to offset an height by a fixed number
+    public ref Height opBinaryAssign (string op : "+=") (ulong offset) return
```

Comment:
Is the `return` attribute right?

Reply:
Yes, because it returns a `ref` to itself (which allows chaining)